routers/routes/routes.go

```diff
@@ -495,6 +495,53 @@ func RegisterRoutes(m *macaron.Macaron) {
 				return
 			}
 
+			if attach.IssueID != 0 {
```

Comment:
How about to move these to a standalone function?

Reply:
I moved the models logic to new `func (a *Attachment) LinkedRepository() (*Repository, UnitType, error)`